src/main/java/io/github/cottonmc/epicurean/item/EpicureanItems.java

```diff
@@ -75,5 +76,6 @@ public static void init() {
 		CompostingChanceRegistry.INSTANCE.add(RAMEN, 1.0f);
 		CompostingChanceRegistry.INSTANCE.add(BEEF_STEW, 1.0f);
 		CompostingChanceRegistry.INSTANCE.add(HAMBURGER, 1.0f);
+		CompostingChanceRegistry.INSTANCE.add(SMOKED_SALMON, 1.0f);
```

Comment:
This should probably be an 0.3f, since that's about the same for normal cooked salmon.